common/src/main/java/com/wynntils/features/user/HadesFeature.java

```diff
@@ -37,14 +37,14 @@ protected void onConfigUpdate(ConfigHolder configHolder) {
             }
             case "shareWithParty" -> {
                 if (shareWithParty) {
-                    Models.PlayerRelations.requestPartyListUpdate();
+                    Models.Party.requestPartyData();
```

Comment:
Since this and requestFriendData are now in separate models, they can be named just `requestData`.